packages/components/src/components/bal-logo/bal-logo.sass

```diff
@@ -1,9 +1,10 @@
 +block(logo)
+  // there is lottie issue, svgs are blurry: https://github.com/airbnb/lottie-web/issues/1976
+  svg
+    transform: unset !important
   position: static
```

Comment:
- [ ] could you move the direct attributes of the block(logo) before the svg child element